Author: Andreas Tille <tille@debian.org>
Last-Update: Wed, 31 Aug 2016 08:41:10 +0200
Origin: https://github.com/doxygen/doxygen/commit/d4ab02c2da7df472bebbf2724419ba00f2de229c
Description: Fix unescaped % and & sign breaking (at least) build of
 cimg (see #819606)

--- a/src/util.cpp
+++ b/src/util.cpp
@@ -6550,6 +6550,8 @@ void filterLatexString(FTextStream &t,co
         case '{':  t << "\\{"; break;
         case '}':  t << "\\}"; break;
         case '_':  t << "\\_"; break;
+        case '%':  t << "\\%"; break;
+        case '&':  t << "\\&"; break;
         case ' ':  if (keepSpaces) t << "~"; else t << ' ';
                    break;
         default:
@@ -6730,6 +6732,8 @@ QCString latexEscapePDFString(const char
       case '\\': t << "\\textbackslash{}"; break;
       case '{':  t << "\\{"; break;
       case '}':  t << "\\}"; break;
+      case '%':  t << "\\%"; break;
+      case '&':  t << "\\&"; break;
       default:
         t << c;
         break;
